modules/javafx.graphics/src/main/java/javafx/application/Platform.java

```diff
@@ -329,6 +331,43 @@ public static void exitNestedEventLoop(Object key, Object rval) {
         Toolkit.getToolkit().exitNestedEventLoop(key, rval);
     }
 
+    /**
+     * Returns a flag indicating whether the key corresponding to {@code keyCode}
+     * is in the locked or "on" state.
+     * {@code keyCode} must be one of: {@link KeyCode#CAPS} or
+     * {@link KeyCode#NUM_LOCK}.
+     * If the underlying system is not able to determine the state of the
+     * specified {@code keyCode}, an empty {@code Optional} is returned.
+     * If the keyboard attached to the system doesn't have the specified key,
+     * {@code Boolean.FALSE} is returned.
+     * This method must be called on the JavaFX Application thread.
+     *
+     * @param keyCode the KeyCode of the lock state to query
+     *
+     * @return the lock state of the key corresponding to {@code keyCode},
+     * or an empty {@code Optional} if the system cannot determine its state
+     *
+     * @throws IllegalArgumentException if {@code keyCode} is not one of the
+     * valid KeyCode values.
```

Comment:
`{@code KeyCode}`

Reply:
In addition, I see that two of the three `@throws` statements end with a period, which I'll remove, since the usual pattern is to omit it.